src/atoms/LogSumExpAtom.jl

```diff
@@ -3,17 +3,26 @@
 # Use of this source code is governed by a BSD-style license that can be found
 # in the LICENSE file or at https://opensource.org/license/bsd-2-clause
 
+"""
+    LogSumExpAtom(x::AbstractExpr, dims::Int = 0)
```

Comment:
should this be `:`? also do we want to canonicalize 0 to `:` or something?

Reply:
Oops yeah. I originally did the `0` think that canonicalized to `:`, then I just made it `Union{Int,Colon}`. Did't update the docstring